src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/config/SwaggerPublisherTest.java

```diff
@@ -14,10 +14,12 @@
 
 class SwaggerPublisherTest extends SpringBootIntegrationBaseTest {
 
+    private static final String SWAGGER_DOCS_VERSION = "/v3/api-docs";
```

Comment:
Can we rename this to be consistent with the Case Event Handler please?  Think it is called `OpenAPIPublisher`.

Reply:
File renamed to OpenAPIPublisherTest